features/checkout/paying_for_order/changing_offline_payment_method_after_order_confirmation.feature

```diff
@@ -0,0 +1,44 @@
+@paying_for_order
+Feature: Changing the offline payment method after order confirmation
+    In order to try different payment methods
+    As a Guest
+    I want to be able to change the method after order confirmation
+
+    Background:
+        Given the store operates on a single channel in "United States"
+        And the store allows paying "Offline"
+        And the store allows paying "Cash on delivery"
+        And the store has a product "PHP T-Shirt" priced at "$19.99"
+        And the store ships everywhere for free
+
+    @ui
+    Scenario: Retrying the payment with different offline payment
+        Given I added product "PHP T-Shirt" to the cart
+        When I complete addressing step with email "john@example.com" and "United States" based billing address
+        And I have proceeded selecting "Free" shipping method
+        And I have proceeded selecting "Cash on delivery" payment method
+        And I have confirmed order
+        And I change payment method to "Offline" after checkout
```

Comment:
This scenarios looks the same as the one with `@api` tag in this feature, and as the real thing that is happening in UI is changing the payment method and then paying I would be for changing this step to sth like:
`I retry the payment with "Offline" payment method`
or go back to what was before:
`I try to pay with with "Offline" payment method`